src/test/java/stringcalculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,52 @@
+package stringcalculator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class StringCalculatorTest {
+    @Test
+    void 빈값입력시예외처리() {
+        String input1 = null;
+        String input2 = "";
+        String input3 = " ";
+        assertAll(
+                () -> assertThatThrownBy(() -> new StringCalculater(input1)).isInstanceOf(IllegalArgumentException.class),
+                () -> assertThatThrownBy(() -> new StringCalculater(input2)).isInstanceOf(IllegalArgumentException.class),
+                () -> assertThatThrownBy(() -> new StringCalculater(input3)).isInstanceOf(IllegalArgumentException.class)
+        );
+
+    }
```

Comment:
junit의 어노테이션을 이용하면 반복적인 표현들을 제거할 수 있습니다.

```suggestion
    @NullSource
    @ValueSource(strings = {"", " "})
    @ParameterizedTest(name = "입력값이 \"{0}\" 이면 예외를 던진다.")
    void 빈값입력시예외처리(String input) {
        assertThatThrownBy(() -> new StringCalculater(input)).isInstanceOf(IllegalArgumentException.class);
    }
```

Reply:
너무 좋네요. 감사합니다. ㅎㅎ